src/openslide-vendor-mirax.c

```diff
@@ -511,7 +511,7 @@ static bool read_nonhier_record(FILE *f,
   }
 
   // read pagesize == 1
-  if (read_le_int32_from_file(f) != 1) {
+  if (read_le_int32_from_file(f) < 1) {
     g_set_error(err, OPENSLIDE_ERROR, OPENSLIDE_ERROR_FAILED,
                 "Expected 1 value");
```

Comment:
The preceding comment and error output no longer match the actual test.

When is this value larger than 1? Is there range of possible values or can it be as much as MAX_UINT? How do we know other code doesn't need changes to handle these larger values.

Reply:
It looks like PR #293 has the same `!=1` to `< 1` change and documents that it seems to occasionally be 2.
That PR also changes some aperio slide parsing, which seems unrelated to the mirax change.